docs/releases/v2022.1.rst

```diff
@@ -0,0 +1,413 @@
+-  RED 15w rev. 1
+
+mpc85xx-p1020
+=============
+
+Extreme Networks
+----------------
+
+-  WS-AP3825i
+
+Device changes
+==============
+
+Renamed Devices
+---------------
+
+-  The ``brcm2708-bcm2708`` ``brcm2708-bcm2709`` ``brcm2708-bcm2710``
+   targets were renamed to ``bcm27xx-bcm2708`` ``bcm27xx-bcm2709`` and
+   ``bcm27xx-bcm2710``.
+-  The GL.iNet GL-AR750S was moved to the ``ath79-nand`` subtarget
+
+Atheros target migration
+------------------------
+
+All Atheros MIPS devices built with the ``ar71xx-generic``,
+``ar71xx-nand`` as well as ``ar71xx-tiny`` were deprecated upstream and
+are therefore not available with Gluon anymore.
+
+Many devices previously built with ``ar71xx-generic`` and
+``ar71xx-nand`` are now available with the ``ath79-generic`` as well as
+``ath79-nand`` target respectively.
+
+Missing devices
+~~~~~~~~~~~~~~~
+
+The following devices have not yet been integrated into Gluons ath79
+targets.
+
+8Devices
+^^^^^^^^
+
+-  Carambola 2
+
+Aerohive
+^^^^^^^^
+
+-  HiveAP 121
+
+Allnet
+^^^^^^
+
+-  ALL0315
+
+Buffalo
+^^^^^^^
+
+-  WZR-HP-G300NH2
+-  WZR-HP-G450H
+
+.. _gl.inet-1:
+
+GL.iNet
+^^^^^^^
+
+-  6408A v1
+
+.. _netgear-1:
+
+NETGEAR
+^^^^^^^
+
+-  WNDR3800
+-  WNDR4300
+-  WNDRMAC
+-  WNDRMAC v2
+
+OCEDO
+^^^^^
+
+-  KOALA
+
+.. _tp-link-3:
+
+TP-Link
+^^^^^^^
+
+-  WR2543
+
+WD
+^^
+
+-  MyNet N600
+-  MyNet N750
+
+ZyXEL
+^^^^^
+
+-  NB6616
+-  NB6716
+
+Removed Devices
+---------------
+
+This list contains devices which do not have enough memory or flash to
+be operated with this Gluon release.
+
+
+D-Link
+~~~~~~
+
+-  DIR-615 (C1, D1, D2, D3, D4, H1)
+
+.. _linksys-1:
+
+Linksys
+~~~~~~~
+
+-  WRT160NL
+
+.. _tp-link-4:
+
+TP-Link
+~~~~~~~
+
+-  TL-MR13U (v1)
+-  TL-MR3020 (v1)
+-  TL-MR3040 (v1, v2)
+-  TL-MR3220 (v1, v2)
+-  TL-MR3420 (v1, v2)
+-  TL-WA701N/ND (v1, v2)
+-  TL-WA730RE (v1)
+-  TL-WA750RE (v1)
+-  TL-WA801N/ND (v1, v2, v3)
+-  TL-WA830RE (v1, v2)
+-  TL-WA850RE (v1)
+-  TL-WA860RE (v1)
+-  TL-WA901N/ND (v1, v2, v3, v4, v5)
+-  TL-WA7210N (v2)
+-  TL-WA7510N (v1)
+-  TL-WR703N (v1)
+-  TL-WR710N (v1, v2)
+-  TL-WR740N (v1, v3, v4, v5)
+-  TL-WR741N/ND (v1, v2, v4, v5)
+-  TL-WR743N/ND (v1, v2)
+-  TL-WR840N (v2)
+-  TL-WR841N/ND (v3, v5, v7, v8, v9, v10, v11, v12)
+-  TL-WR841N/ND (v1, v2)
+-  TL-WR843N/ND (v1)
+-  TL-WR940N (v1, v2, v3, v4, v5, v6)
+-  TL-WR941ND (v2, v3, v4, v5, v6)
+-  TL-WR1043N/ND (v1)
+-  WDR4900
+
+.. _ubiquiti-2:
+
+Ubiquiti
+~~~~~~~~
+
+-  AirGateway
+-  AirGateway Pro
+-  AirRouter
+-  Rocket
+-  Bullet
+-  Nanostation XW
+-  LS-SR71
+-  Bullet
+
+Unknown
+~~~~~~~
+
+-  A5-V11
+
+VoCore
+~~~~~~
+
+-  VoCore (8M, 16M)
+
+Features
+========
+
+Wireguard
+---------
+
+ToDo: LEMOER OR AIYION
+
+fastd L2TP
+----------
+
+fastd can now act as a connection broker for unencrypted L2TP-based
+tunneling within Gluons mesh-vpn framework.
+
+In addition to a sufficient configured fastd-based VPN server, this
+requires adding the ``null@l2tp`` method to the fastd site.
```

Comment:
- `null@l2tp` needs to be at the **top** of the list
- `mesh-vpn-fastd-l2tp` needs to be added within the site.mk

Maybe it's better to just link to https://github.com/freifunk-gluon/gluon/blob/master/docs/features/vpn.rst#methods. idk

Reply:
Now it sounds a bit like Wireguard is for performance, but fastd L2TP is not? Would it make sense to add a common paragraph to say "Gluon got two new VPN methods which have increased performance: Wireguard and fastd L2TP. Bc. both fully in kernel space etc." And then sub sections which describe their differences?

Also, currently the Wireguard section mentions VPN only at the end. And it might sound as if were increasing performance for any encrypted connection?

Reply:
I've fixed the link to the respective wiki page

@T-X This sounds like an improvement to the VPN wiki page and not the release notes, so I don't think this is actionable for this PR?

Reply:
> 20:39:00 <               T_X > blocktrr1: thanks, the "Known issues" section - and everything else in the release notes, sounds good to me!
> 20:39:30 <               T_X > also thanks for adding a note for the new OpenWrt / Linux kernel / wireless backports version in this new release
>